crates/oxc_linter/src/rules/eslint/no_console.rs

```diff
@@ -88,6 +95,47 @@ impl Rule for NoConsole {
     }
 }
 
+fn remove_console<'c, 'a: 'c>(
+    fixer: RuleFixer<'c, 'a>,
+    ctx: &'c LintContext<'a>,
+    node: &AstNode<'a>,
+) -> RuleFix<'a> {
+    let mut node_to_delete = node;
+    for parent in ctx.nodes().iter_parents(node.id()).skip(1) {
+        match parent.kind() {
+            AstKind::ParenthesizedExpression(_)
+            | AstKind::ExpressionStatement(_)
+            => node_to_delete = parent,
+            AstKind::IfStatement(_)
+            | AstKind::WhileStatement(_)
+            | AstKind::ForStatement(_)
+            | AstKind::ForInStatement(_)
+            | AstKind::ForOfStatement(_)
+            | AstKind::ArrowFunctionExpression(_) => {
+                return fixer.replace(node_to_delete.span(), "{}")
+            }
+            // Arrow function AST nodes do not say whether they have brackets or
```

Comment:
We can check:
https://github.com/oxc-project/oxc/blob/d3ffe0f9fff62e0bddad52f13e1ad7c196f4d7cf/crates/oxc_ast/src/ast/js.rs#L1560-L1561